packages/runtime/src/next/client.ts

```diff
@@ -129,6 +129,12 @@ export type Sitemap = {
   }[]
 }[]
 
+type GetPageResult = {
+  path: string
+  locale: string
+  localized: { path: string; locale: string }[]
```

Comment:
I'm not 100% sure what to name this field, it could be `localized`, `alternate`, or something else.

Reply:
I lean towards `alternate`

Also, can we please rename the field `pathname` instead of `path`? URL paths include search params but for Makeswift pages what identifies them is their pathname

Reply:
Updated to `alternate` and `pathname`